### DISCUSSION TITLE
how to remove  load button form excalidraw

### DISCUSSION THREAD

**Discussion by Sohailamjad1** [excalidraw/excalidraw#4962] [Q&A]
Hi ..Iam trying to remove load icon which is comes next to export icon ..could you please help on this....Thanks

**Comment by ad1992** [comment]
You can use [UIOptions](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#uioptions) and set `loadScene` to false. Please go through the documentation, Thanks!

**Reply by Sohailamjad1** [reply]
hey ..can you tell me where i need to add the loadScene to false?

**Reply by Sohailamjad1** [reply]
Working ,Thanks
